spec/models/concerns/interval_spec.rb

```diff
@@ -123,6 +123,26 @@
         expect(DummyMentor).to have_received(:starting_tomorrow_or_after).once
       end
     end
+
+    describe ".closest_to" do
+      it "returns a SQL relation selecting the closest record" do
+        date = Date.new(2023, 12, 10)
+        sql = DummyMentor.closest_to(date).to_sql
+
+        expect(sql).to include("ORDER BY LEAST")
+        expect(sql).to include('ABS("mentor_at_school_periods".started_on - DATE \'2023-12-10\')')
```

Comment:
Can we change these tests to use actual data/models? I think asserting on the raw SQL for these quite complex scopes is probably a bit error prone 🤔